src/table/components/__tests__/TableTotals.spec.tsx

```diff
@@ -1,32 +1,32 @@
 import React from 'react';
-import { render, fireEvent } from '@testing-library/react';
+import { render, fireEvent, Matcher } from '@testing-library/react';
+import { stardust } from '@nebula.js/stardust';
 import TableTotals from '../TableTotals';
 import { TableContextProvider } from '../../context';
 import { getTotalPosition } from '../../../handle-data';
 import * as handleKeyPress from '../../utils/handle-key-press';
 import * as handleAccessibility from '../../utils/handle-accessibility';
 import { generateLayout } from '../../../__test__/generate-test-data';
+import { TableData, ExtendedTheme, TableLayout } from '../../../types';
 
 describe('<TableTotals />', () => {
-  const rootElement = {};
-  let tableData;
-  let theme;
-  let layout;
-  let selectionsAPI;
-  let keyboard;
-  let translator;
+  const rootElement = {} as HTMLElement;
+  let tableData: TableData;
+  let theme: ExtendedTheme;
+  let layout: TableLayout;
+  let selectionsAPI: stardust.ObjectSelections;
+  let keyboard: stardust.Keyboard;
 
-  const renderTableTotals = (cellCoordMock) =>
+  const renderTableTotals = (cellCoordMock?: [number, number]) =>
     render(
       <TableContextProvider selectionsAPI={selectionsAPI} cellCoordMock={cellCoordMock}>
-        {getTotalPosition(layout) === 'top' && (
+        {((getTotalPosition(layout) === 'top') as unknown as HTMLElement) && (
```

Comment:
It's not a good idea to set the string type as HTMLElement but I didn't have another option in my mind to fix the type conflict here... the total position check here makes the problem

Reply:
How about
```
{getTotalPosition(layout) === 'top' ? (
          <TableTotals
            rootElement={rootElement}
            tableData={tableData}
            theme={theme}
            layout={layout}
            keyboard={keyboard}
          />
        ) : (
          <></>
        )}
```

Reply:
yes it makes sense and works properly, thanks!